packages/kit/src/runtime/client/client.js

```diff
@@ -179,7 +179,9 @@ let target;
 export let app;
 
 /** @type {Record<string, any>} */
-export const remote_responses = __SVELTEKIT_PAYLOAD__.data ?? {};
+// avoid referencing `__SVELTEKIT_PAYLOAD__` because it will be undefined unless this module was loaded by Vite
```

Comment:
I don't fully understand yet why Vitest wouldn't load this through Vite module loader yet. If anything, this comment needs to be enhanced a bit to clarify this.
I also don't fully understand why this global is causing problems but the other globals in this file do not - would they also cause problems if we started using them for more than booleans, i.e. property access?

Reply:
You're right. I was completely misunderstanding the issue. Vite does perform the global variable replacements. It's just that our `globalThis.__sveltekit_dev` object is `undefined` unless it's defined by our server rendered page.

So the only issue was accessing a property on an undefined object. I'll just add an optional chain.